controllers/homeRoutes.js

```diff
@@ -64,16 +65,28 @@ router.get('/space/:space_id/access', withNoMembership, withAuth, async (req, re
     res.status(400).json(err);
     console.log(err);
   }
+});
+
+// Get idea create page
+router.get('/space/:space_id/ideas', withAuth, async (req, res) => {
```

Comment:
Add the `withApprovedMembership` authentication helper. See `/space/:space_id` route

Reply:
Done